src/components/grid-aware/Footer/Footer.module.css

```diff
@@ -0,0 +1,257 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-rows: 120px 380px 78px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    text-align: center;
+  }
+
+  .gridParent {
+    column-gap: var(--grid-column-gap);
+    display: grid;
+    grid-template-rows: 190px auto auto;
+  }
+}
+
+/* Top Row */
+
+.gridTopRow {
+  align-content: center;
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  grid-row: 1;
+  justify-content: center;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  align-self: center;
+  display: flex;
+  justify-content: center;
+}
+
+.subscribeMessage {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  padding-right: 20px;
+  width: 340px;
+}
+
+.gridTopRow a {
+  padding: 14px 40px;
+}
+
+.gridTopRow a:hover {
+  top: 0;
+}
+
+.formInput {
+  display: flex;
+  flex-direction: row;
+}
+
+.formInput input {
+  border: 0;
+  color: var(--color-black);
+  font: var(--font-body-small);
+  padding: 16px 0 15px 17px;
```

Comment:
Even if there isn't guidance here on the right padding, I don't think we should set it to zero. That's going to make the element look weird and imbalanced if you manage to fill up the whole text box. The natural thing to do is to set equal left and right padding.

Reply:
ok